### ISSUE TITLE
2025-01-14, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#51]
# Today's Arxiv Papers

(1) [WebWalker: Benchmarking LLMs in Web Traversal](http://arxiv.org/abs/2501.07572v1), 2025-01-13

(2) [SST-EM: Advanced Metrics for Evaluating Semantic, Spatial and Temporal Aspects in Video Editing](http://arxiv.org/abs/2501.07554v1), 2025-01-13

(3) [Imagine while Reasoning in Space: Multimodal Visualization-of-Thought](http://arxiv.org/abs/2501.07542v1), 2025-01-13

(4) [ML Mule: Mobile-Driven Context-Aware Collaborative Learning](http://arxiv.org/abs/2501.07536v1), 2025-01-13

(5) [Investigating Large Language Models in Inferring Personality Traits from User Conversations](http://arxiv.org/abs/2501.07532v1), 2025-01-13

(6) [Evaluating Agent-based Program Repair at Google](http://arxiv.org/abs/2501.07531v1), 2025-01-13

(7) [RadAlign: Advancing Radiology Report Generation with Vision-Language Concept Alignment](http://arxiv.org/abs/2501.07525v1), 2025-01-13

(8) [Parallel Key-Value Cache Fusion for Position Invariant RAG](http://arxiv.org/abs/2501.07523v1), 2025-01-13

(9) [Exploring and Mitigating Adversarial Manipulation of Voting-Based Leaderboards](http://arxiv.org/abs/2501.07493v1), 2025-01-13

(10) [TiEBe: A Benchmark for Assessing the Current Knowledge of Large Language Models](http://arxiv.org/abs/2501.07482v1), 2025-01-13

(11) [A Survey of Embodied AI in Healthcare: Techniques, Applications, and Opportunities](http://arxiv.org/abs/2501.07468v1), 2025-01-13

(12) [Understanding and Benchmarking Artificial Intelligence: OpenAI's o3 Is Not AGI](http://arxiv.org/abs/2501.07458v1), 2025-01-13

(13) [Enhancing LLM's Ability to Generate More Repository-Aware Unit Tests Through Precise Contextual Information Injection](http://arxiv.org/abs/2501.07425v1), 2025-01-13

(14) [Initial Findings on Sensor based Open Vocabulary Activity Recognition via Text Embedding Inversion](http://arxiv.org/abs/2501.07408v1), 2025-01-13

(15) [Emergent effects of scaling on the functional hierarchies within large language models](http://arxiv.org/abs/2501.07359v1), 2025-01-13

(16) [Occamy: A 432-Core Dual-Chiplet Dual-HBM2E 768-DP-GFLOP/s RISC-V System for 8-to-64-bit Dense and Sparse Computing in 12nm FinFET](http://dx.doi.org/10.1109/JSSC.2025.3529249), 2025-01-13

(17) [Foundation Models at Work: Fine-Tuning for Fairness in Algorithmic Hiring](http://arxiv.org/abs/2501.07324v1), 2025-01-13

(18) [FinerWeb-10BT: Refining Web Data with LLM-Based Line-Level Filtering](http://arxiv.org/abs/2501.07314v1), 2025-01-13

(19) [The Lessons of Developing Process Reward Models in Mathematical Reasoning](http://arxiv.org/abs/2501.07301v1), 2025-01-13

(20) [GestLLM: Advanced Hand Gesture Interpretation via Large Language Models for Human-Robot Interaction](http://arxiv.org/abs/2501.07295v1), 2025-01-13

(21) [LLM-Net: Democratizing LLMs-as-a-Service through Blockchain-based Expert Networks](http://arxiv.org/abs/2501.07288v1), 2025-01-13

(22) [Lifelong Learning of Large Language Model based Agents: A Roadmap](http://arxiv.org/abs/2501.07278v1), 2025-01-13

(23) [Bridging Smart Meter Gaps: A Benchmark of Statistical, Machine Learning and Time Series Foundation Models for Data Imputation](http://arxiv.org/abs/2501.07276v1), 2025-01-13

(24) [Transforming Role Classification in Scientific Teams Using LLMs and Advanced Predictive Analytics](http://arxiv.org/abs/2501.07267v1), 2025-01-13

(25) [Large Language Models: New Opportunities for Access to Science](http://arxiv.org/abs/2501.07250v1), 2025-01-13

(26) [Lessons From Red Teaming 100 Generative AI Products](http://arxiv.org/abs/2501.07238v1), 2025-01-13

(27) [Breaking Memory Limits: Gradient Wavelet Transform Enhances LLMs Training](http://arxiv.org/abs/2501.07237v1), 2025-01-13

(28) [Touched by ChatGPT: Using an LLM to Drive Affective Tactile Interaction](http://arxiv.org/abs/2501.07224v1), 2025-01-13

(29) [FaceOracle: Chat with a Face Image Oracle](http://arxiv.org/abs/2501.07202v1), 2025-01-13

(30) [Pre-Trained Large Language Model Based Remaining Useful Life Transfer Prediction of Bearing](http://arxiv.org/abs/2501.07191v1), 2025-01-13

(31) [Unveiling Code Clone Patterns in Open Source VR Software: An Empirical Study](http://arxiv.org/abs/2501.07165v1), 2025-01-13

(32) [FlexQuant: Elastic Quantization Framework for Locally Hosted LLM on Edge Devices](http://arxiv.org/abs/2501.07139v1), 2025-01-13

(33) [LLM360 K2: Scaling Up 360-Open-Source Large Language Models](http://arxiv.org/abs/2501.07124v1), 2025-01-13

(34) [How GPT learns layer by layer](http://arxiv.org/abs/2501.07108v1), 2025-01-13

(35) [ADKGD: Anomaly Detection in Knowledge Graphs with Dual-Channel Training](http://arxiv.org/abs/2501.07078v1), 2025-01-13

(36) [Value Compass Leaderboard: A Platform for Fundamental and Validated Evaluation of LLMs Values](http://arxiv.org/abs/2501.07071v1), 2025-01-13

(37) [Enhancing Image Generation Fidelity via Progressive Prompts](http://arxiv.org/abs/2501.07070v1), 2025-01-13

(38) [Logic Meets Magic: LLMs Cracking Smart Contract Vulnerabilities](http://arxiv.org/abs/2501.07058v1), 2025-01-13

(39) [PoAct: Policy and Action Dual-Control Agent for Generalized Applications](http://arxiv.org/abs/2501.07054v1), 2025-01-13

(40) [ROSAnnotator: A Web Application for ROSBag Data Analysis in Human-Robot Interaction](http://arxiv.org/abs/2501.07051v1), 2025-01-13

(41) [Unveiling the Potential of Text in High-Dimensional Time Series Forecasting](http://arxiv.org/abs/2501.07048v1), 2025-01-13

(42) [A Proposed Large Language Model-Based Smart Search for Archive System](http://arxiv.org/abs/2501.07024v1), 2025-01-13

(43) [LEO: Boosting Mixture of Vision Encoders for Multimodal Large Language Models](http://arxiv.org/abs/2501.06986v1), 2025-01-13

(44) [Combining LLM decision and RL action selection to improve RL policy for adaptive interventions](http://arxiv.org/abs/2501.06980v1), 2025-01-13

(45) [How is Google using AI for internal code migrations?](http://arxiv.org/abs/2501.06972v1), 2025-01-12

(46) [Enhancing Patient-Centric Communication: Leveraging LLMs to Simulate Patient Perspectives](http://arxiv.org/abs/2501.06964v1), 2025-01-12

(47) [Comparison of Autoencoders for tokenization of ASL datasets](http://arxiv.org/abs/2501.06942v1), 2025-01-12

(48) [Harnessing Large Language Models for Disaster Management: A Survey](http://arxiv.org/abs/2501.06932v1), 2025-01-12

(49) [Monolithic 3D FPGAs Utilizing Back-End-of-Line Configuration Memories](http://arxiv.org/abs/2501.06921v1), 2025-01-12

(50) [Risk-Averse Finetuning of Large Language Models](http://arxiv.org/abs/2501.06911v1), 2025-01-12

(51) [Transfer Learning of Tabular Data by Finetuning Large Language Models](http://arxiv.org/abs/2501.06863v1), 2025-01-12

(52) [A Comprehensive Evaluation of Large Language Models on Mental Illnesses in Arabic Context](http://arxiv.org/abs/2501.06859v1), 2025-01-12

(53) [SPAM: Spike-Aware Adam with Momentum Reset for Stable LLM Training](http://arxiv.org/abs/2501.06842v1), 2025-01-12

(54) [An efficient approach to represent enterprise web application structure using Large Language Model in the service of Intelligent Quality Engineering](http://arxiv.org/abs/2501.06837v1), 2025-01-12

(55) [X-LeBench: A Benchmark for Extremely Long Egocentric Video Understanding](http://arxiv.org/abs/2501.06835v1), 2025-01-12

(56) [LLMs Model Non-WEIRD Populations: Experiments with Synthetic Cultural Agents](http://arxiv.org/abs/2501.06834v1), 2025-01-12

(57) [GeoPix: Multi-Modal Large Language Model for Pixel-level Image Understanding in Remote Sensing](http://arxiv.org/abs/2501.06828v1), 2025-01-12

(58) [Leveraging Taxonomy and LLMs for Improved Multimodal Hierarchical Classification](http://arxiv.org/abs/2501.06827v1), 2025-01-12

(59) [Event Argument Extraction with Enriched Prompts](http://arxiv.org/abs/2501.06825v1), 2025-01-12

(60) [A Study on Educational Data Analysis and Personalized Feedback Report Generation Based on Tags and ChatGPT](http://arxiv.org/abs/2501.06819v1), 2025-01-12

(61) [MPCache: MPC-Friendly KV Cache Eviction for Efficient Private Large Language Model Inference](http://arxiv.org/abs/2501.06807v1), 2025-01-12

(62) [Unifying Two Types of Scaling Laws from the Perspective of Conditional Kolmogorov Complexity](http://arxiv.org/abs/2501.06802v1), 2025-01-12

(63) [Bridging the Fairness Gap: Enhancing Pre-trained Models with LLM-Generated Sentences](http://arxiv.org/abs/2501.06795v1), 2025-01-12

(64) [Eliza: A Web3 friendly AI Agent Operating System](http://arxiv.org/abs/2501.06781v1), 2025-01-12

(65) [VidChain: Chain-of-Tasks with Metric-based Direct Preference Optimization for Dense Video Captioning](http://arxiv.org/abs/2501.06761v1), 2025-01-12

(66) [Hierarchical Divide-and-Conquer for Fine-Grained Alignment in LLM-Based Medical Evaluation](http://arxiv.org/abs/2501.06741v1), 2025-01-12

(67) [Hold On! Is My Feedback Useful? Evaluating the Usefulness of Code Review Comments](http://arxiv.org/abs/2501.06738v1), 2025-01-12

(68) [ZOQO: Zero-Order Quantized Optimization](http://arxiv.org/abs/2501.06736v1), 2025-01-12

(69) [Measuring the Robustness of Reference-Free Dialogue Evaluation Systems](http://arxiv.org/abs/2501.06728v1), 2025-01-12

(70) [ZNO-Eval: Benchmarking reasoning capabilities of large language models in Ukrainian](http://dx.doi.org/10.15276/ict.01.2024.27), 2025-01-12

(71) [MiniRAG: Towards Extremely Simple Retrieval-Augmented Generation](http://arxiv.org/abs/2501.06713v1), 2025-01-12

(72) [Mell: Memory-Efficient Large Language Model Serving via Multi-GPU KV Cache Management](http://arxiv.org/abs/2501.06709v1), 2025-01-12

(73) [AIOpsLab: A Holistic Framework to Evaluate AI Agents for Enabling Autonomous Clouds](http://arxiv.org/abs/2501.06706v1), 2025-01-12

(74) [Fine-tuning ChatGPT for Automatic Scoring of Written Scientific Explanations in Chinese](http://arxiv.org/abs/2501.06704v1), 2025-01-12

(75) [Large Language Models, Knowledge Graphs and Search Engines: A Crossroads for Answering Users' Questions](http://arxiv.org/abs/2501.06699v1), 2025-01-12

(76) [DVM: Towards Controllable LLM Agents in Social Deduction Games](http://arxiv.org/abs/2501.06695v1), 2025-01-12

(77) [TAPO: Task-Referenced Adaptation for Prompt Optimization](http://arxiv.org/abs/2501.06689v1), 2025-01-12

(78) [Generative AI in Education: From Foundational Insights to the Socratic Playground for Learning](http://arxiv.org/abs/2501.06682v1), 2025-01-12

(79) [Application of Vision-Language Model to Pedestrians Behavior and Scene Understanding in Autonomous Driving](http://arxiv.org/abs/2501.06680v1), 2025-01-12
